_docs/welcome/notes/0.4.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Istio 0.4
+
+order: 97
+
+layout: docs
+type: markdown
+---
+
+## General
+
+This release has only got a few weeks worth of changes, as we stabilize our monthly release process. 
+In addition to the usual pile of bug fixes and perf improvements, this release includes:
+
+- **Cloud Foundry**. Added support for the [Cloud Foundry](https://www.cloudfoundry.org) platform to Pilot, making it
```

Comment:
is this ready ? I don't see any instructions ? maybe say something like "initial support..."